### ISSUE TITLE
Revit exports invalid IfcFurnitureType

### ISSUE THREAD

**Issue by Moult** [Autodesk/revit-ifc#395]
The AssemblyPlace attribute is mandatory but Revit seems to leave it blank instead of choosing a default like NOTDEFINED.

**Comment by JOuellette-Autodesk** [comment]
@Moult could you tell us which version of Revit and the IFC tool you are seeing this?

**Comment by Moult** [comment]
Revit 2021 latest version of the Revit exporter.

**Comment by Moult** [comment]
Not sure if this should just be marked as feedback - it causes basic IFC validation to fail.

**Comment by AngelVelezSosa** [comment]
I agree, @JOuellette-Autodesk can you file a problem report for this also?

**Comment by JOuellette-Autodesk** [comment]
@Moult "Feedback" is a temporary label for my purposes of triage. I use it when I ask for more info from the submitter. I remove it when I get the answers.

**Comment by JOuellette-Autodesk** [comment]
@Moult BTW, Revit doesn't even support IFC4.3 yet. Are you submitting an issue against IFC4.0.2.1?

**Comment by JOuellette-Autodesk** [comment]
https://jira.autodesk.com/browse/REVIT-188936

**Comment by Moult** [comment]
Not sure where IFC4.3 is relevant here, This bug is relevant to IFC2X3 and IFC4, regardless of MVD.

**Comment by JOuellette-Autodesk** [comment]
@Moult Sample file? I want to test this in 2023
